code/modules/mob/living/carbon/xenomorph/Powers.dm

```diff
@@ -2192,13 +2196,15 @@
 	to_chat(src, "<span class='xenowarning'>Your stinger injects your victim with neurotoxin and larval growth serum!</span>")
 	playsound(H, 'sound/effects/spray3.ogg', 15, 1)
 	playsound(H, pick('sound/voice/alien_drool1.ogg', 'sound/voice/alien_drool2.ogg'), 15, 1)
+	overdose_check(H)
```

Comment:
Repeated?

Reply:
Every iteration of injection checks for OD so the xeno knows when to pack off.

Reply:
Sure, but you have two checks:
```
overdose_check(H)
overdose_check(H, "xeno_growthtoxin")
```
Intended?

Reply:
No need to check on `xeno_toxin` here, I'd say.

Reply:
Needed to alert the Defiler if the initial injection is enough to OD.